addons/website_profile/static/src/js/website_profile.js

```diff
@@ -25,12 +25,23 @@ publicWidget.registry.websiteProfile = publicWidget.Widget.extend({
     _onSendValidationEmailClick: function (ev) {
         ev.preventDefault();
         var $element = $(ev.currentTarget);
+
+        if (this.emailRequested) {
+            return;
+        }
+
+        $element[0].style.cursor = "progress";
+        this.emailRequested = true;
```

Comment:
I try to always do my latch like this because it minimizes the risk of error and makes it more legible. What do you think?

```suggestion
        if (this.emailRequested) {
            return;
        }
        this.emailRequested = true;
        
        var $element = $(ev.currentTarget);
        $element[0].style.cursor = "progress";
```